EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -58,7 +58,9 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usererrorlist = config_.getParameter<std::vector<int> > ("UserErrorList");
   }
   tFEDRawDataCollection = consumes <FEDRawDataCollection> (config_.getParameter<edm::InputTag>("InputLabel"));
-
+  
+  theBadPixelFEDChannelsLabel = consumes<PixelFEDChannelCollection>(edm::InputTag("mix"));
```

Comment:
It is run in the reconstruction job. My point is that the product label of sim digis is different in classical mixing (`mix`) and premixing (`mixData`) workflows.

Reply:
I see... so the DigiToRaw module must be configured differently for classical mixing than for premixing. If/when we implement passing the bad channel information through the DigiToRaw --> RawToDigi detour then this will have a natural solution by using the same Tag. Need to keep in mind to cross-check it, though.

Reply:
> so the DigiToRaw module must be configured differently for classical mixing than for premixing.

Indeed, done here

https://github.com/cms-sw/cmssw/blob/d74dd1812ca6cb948355c4796b5c39a8500f4ad1/Configuration/StandardSequences/python/DigiToRawDM_cff.py#L10

Reply:
done